examples/gradle-example/src/main/java/com/autonomic/tmc/example/auth/GradleAuthExample.java

```diff
@@ -50,36 +49,43 @@ public void run(String... args) {
     }
 
     private void run() {
-        try {
-            TokenSupplier tokenSupplier;
+        TokenSupplier tokenSupplier = null;
 
-            //#Example 1: REST Authentication token for Production
+        //#Example 1: REST Authentication token for Production
+        try {
             tokenSupplier = createTokenSupplier(clientId, clientSecret);
             printToken(tokenSupplier, "with default URL");
+        } catch (BaseSdkException e) {
+            LOGGER.log(Level.SEVERE, "Generic message, Something went wrong: ", e);
```

Comment:
Suggestion:  We have the opportunity here to tell which example failed.  Should we?  If you like this idea, please make this change is made in both example classes (Maven and Gradle) for all three examples.

```suggestion
            LOGGER.log(Level.SEVERE, "Example 1: REST Authentication token for Production. Something went wrong: ", e);
```